index.html

```diff
@@ -209,54 +209,40 @@ <h2 class="gray-underline">How We Work</h2>
 <div class="section">
     <div class="container">
         <div class="title-header">
-            <h2>Our values</h2>
+          <h2>Software Craftsmanship</h2>
+          <p>Software Craftsmanship is at the heart of everything we do. We constantly hone our skills by teaching and learning from others within our company and the wider software development community.</p>
         </div>
-        <div class="row content-boxes">
-            <div class="col-md-4 md-margin-bottom-30">
-                <i class="fa fa-trophy fa-3x"></i>
-                <div class="content-box">
-                    <h3 class="heading-sm">QUALITY</h3>
-                    <p>We don't negotiate on quality. Software must deliver more value with time and not be a source of increasing pain and cost.</p>
+            <div class="row news-v1">
+                <div class="col-md-4 md-margin-bottom-40">
+                    <div class="news-v1-in">
+                        <img class="img-responsive" src="/assets/img/custom/home/the-software-craftsman-book.jpg" alt="">
+                        <h3><a href="#">The Software Craftsman: by our Co-founder Sandro Mancuso</a></h3>
```

Comment:
Do we really want to scroll to the top of the page when the link is clicked?

In case we don't, the same thing happens in the other columns' anchors